hera_cal/vis_clean.py

```diff
@@ -688,8 +688,10 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode,
                                 flgs[i] = True
 
             filtered_model[k] = mdl
+            filtered_model[k][flgs] = 0.
             filtered_resid[k] = res
-            filtered_data[k] = mdl + res * fw
+            filtered_resid[k][flgs | f] = 0.
```

Comment:
perhaps to preserve continuity from how we've been doing it, you can use
```
filtered_resid[k] = res * fw
filtered_resid[k][flgs] = 0.0
```
which sets the residual at flagged channels to zero

Reply:
I've adopted this convention.